src/Illuminate/Foundation/Console/stubs/echo-bootstrap-js.stub

```diff
@@ -4,4 +4,4 @@
  * allow your team to quickly build robust real-time web applications.
  */
 
-import './echo';
+import './echo.js';
```

Comment:
What does this line change fix?

Reply:
Mistakenly pushed to this PR. Since it's related to broadcast installation, I kept the commit. The bug is explained below:

**Problem**: The installation keeps inserting import of echo in bootrap.js

**Buggy Code:**
`if (! str_contains($bootstrapScript, 'echo.js'))`

The import checker checks for echo.js, that's okay but the stub not follows the checker strategy, which makes the checker always return false

Thus, repeating import of echo.js for every `install:broadcasting` trial